July 2026 newsletter (#1056)
RSE-Sheffield/RSE-Sheffield.github.io: _newsletters/2026-07-01-newsletter.md

---
layout: post
title: July 2026 Newsletter
editor: Martin Dyer
slug: 2026-07-01-newsletter
date: 2026-07-01T12:00:00Z+0100
tags:
category:
link:
description:
type: text
---

## The University of Sheffield Research Software Engineering Community Newsletter July 2026

Welcome to the July 2026 newsletter for the research software community at The University of Sheffield, featuring
news, opportunities, events and training for you.

<!-- --- -->

### News

> - The **RSE Call for Proposals 2026** is still open. 6 months of RSE collaboration at 0.5 FTE is available for four projects, two of which are funded by the Henry Royce Institute and will support projects from Materials Science.

Reviewer: ```suggestion
- The **RSE Call for Proposals 2026** is still open. 6 months of RSE collaboration at 0.5 FTE is available for four projects, two of which are funded by the Henry Royce Institute and will support projects within the field of Materials Science.
```

From implies from a specific department, whereas Romain clarified the projects self identify as touching materials science.

cc @ns-rse

  - [Full details](https://rse.shef.ac.uk/collaboration/RSEtime/2026/)
  - Submission deadline: **2026-07-14 17:00**

---

### Events

<!-- Nothing scheduled but left as a place-holder should this be used as a template for next newsletter -->

<!-- ### RSE lunchtime seminars -->

<!-- Members of Sheffield RSE will be hosting a new small seminar series in the new year, with **catered lunches**! -->

<!-- You can sign up on [MyDevelopment]() (requires sign in to MUSE). -->

#### Upcoming Events in Sheffield

- [AI for Research Day 2026](https://sites.google.com/sheffield.ac.uk/ai-research-day), held under the theme _AI for Research That Matters_.
  - Date: **2026-10-01**
  - Venue: INOX, Sheffield Students’ Union

>   - Led by the Centre for Machine Intelligence, the day shows how researchers across every discipline are putting AI to work. It's built for people who are curious about AI but haven't yet brought it into their own research — with plenty for those already using these tools, too. You'll leave with a clearer sense of where AI fits in your work, and the practical experience to take the next step.

Reviewer 2: Increasingly its going to be a case of! (This is a cynical and sarcastic comment and can be completely ignored)
 
```suggestion
  - Led by the Centre for Machine Intelligence, the day shows how researchers across every discipline are putting AI to work. It's built for people who are curious about AI but haven't yet bought it into their own research — with plenty for those already using these tools, too. You'll leave with a clearer sense of where AI fits in your work, and the practical experience to take the next step.
```

- [How hidden contributions power modern research](https://www.nature.com/articles/d41586-026-01260-x) - The people who work behind the scenes to keep research moving say that there should be more recognition for their roles.

#### Blogs

- [Who Will Keep Research Data Infrastructure Open and Running?](https://issues.org/open-data-research-%20infrastructure-gibson-thaney/) - The scientific community must consider the longevity of open research infrastructure—why it might fail and how to prevent it.

- [Inclusivity and Accessibility in Recruitment](https://www.software.ac.uk/guide/inclusivity-and-accessibility-recruitment) - Software Sustainability Institute

- [ConveRSE at Collaborations Workshop 2026](https://www.software.ac.uk/blog/converse-collaborations-workshop-2026) - Software Sustainability Institute

#### Papers

- [Recommendations from a Fellowship Programme for Advocates of Good Software Practices in Research](https://openresearchsoftware.metajnl.com/articles/10.5334/jors.616) - Journal of Open Research Software

#### Pre-Prints

> - [Coding is not enough](https://zenodo.org/records/20666679) - Romain Thomas & Edwin Brown (Research Software Engineering, University of Sheffield)

Reviewer 2: 👍

- [Podcast: What happened to agile development?](https://codeforthought.buzzsprout.com/1326658/episodes/19292687-en-what-happened-to-agile-development-a-review-with-dave-thomas) - A review with Dave Thomas - Code for Thought

- [Podcast: ByteSized: Security by Design](https://codeforthought.buzzsprout.com/1326658/episodes/19335251-en-bytesized-security-by-design-secure-data-principles-for-digital-research-with-e-sheppard-m-hardaker) - Secure Data Principles for Digital Research

<!-- #### Books -->

#### Training Materials

- [Creating Community Playbooks (PBK)](https://www.cscce.org/trainings/pbk/) - Center for Scientific Collaboration and Community Engagement

---

### Opportunities

- UKRI are looking for new members to join the UKRI Advisory Group for Digital Research Infrastructure (AGD). [Full details](https://www.ukri.org/who-we-are/work-for-us/join-an-advisory-committee-panel-or-network/digital-research-infrastructure-advisory-group-member-vacancies/)
  - Closing date: **2026-08-07**

- [Green RSE Award 2026](https://www.software.ac.uk/news/green-rse-award-returns-2026) nominations are open.

>   - Closing date: **2026-07-13**

Reviewer: Ordered with soonest deadline first

```suggestion
- [Green RSE Award 2026](https://www.software.ac.uk/news/green-rse-award-returns-2026) nominations are open.
  - Closing date: **2026-07-13**
  
- UKRI are looking for new members to join the UKRI Advisory Group for Digital Research Infrastructure (AGD). [Full details](https://www.ukri.org/who-we-are/work-for-us/join-an-advisory-committee-panel-or-network/digital-research-infrastructure-advisory-group-member-vacancies/)
  - Closing date: **2026-08-07**
```
